src/config/index.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  project: {
+    name: 'illustratedlife',
```

Comment:
Is the plan to override this for each talk project we want to archive? Should we use ENV vars in here to inject via the build scripts?

Reply:
The deploy target in `package.json` will need to be changed too.

Reply:
I think you're right that the final version will get site name, prefix and deploy target from ENV.